checker/src/main/java/org/checkerframework/checker/index/lowerbound/LowerBoundTransfer.java

```diff
@@ -693,6 +699,36 @@ private AnnotationMirror addAnnotationForLiteralRemainder(int val) {
         return UNKNOWN;
     }
 
+    @Override
+    protected void addInformationFromPreconditions(
+            CFStore info,
+            AnnotatedTypeFactory factory,
+            UnderlyingAST.CFGMethod method,
+            MethodTree methodTree,
+            ExecutableElement methodElement) {
+        super.addInformationFromPreconditions(info, factory, method, methodTree, methodElement);
+
+        List<? extends VariableTree> paramTrees = methodTree.getParameters();
+
+        for (VariableTree variableTree : paramTrees) {
+            if (variableTree.getType().toString().equalsIgnoreCase("char")) {
```

Comment:
This is a poor way to check that the variable is a `char`, but couldn't figure out another way.

Reply:
You can use `TreeUtils.typeOf(variableTree).getKind() == TypeKind.CHAR` instead